docs/en/entity-onboarding.rst

```diff
@@ -0,0 +1,1095 @@
+
+
+
+**Step 1 - Onboarding Request Submission**: Federation Entity initiates the onboarding process by submitting a technical registration request including the following information.
+
+.. list-table:: Federation Onboarding Technical Request Information
+   :class: longtable
+   :header-rows: 1
+   :widths: 30 70
+
+   * - **Technical Information Category**
+     - **Requirements and Description**
+   * - **Federation Entity Identifier**
+     - **REQUIRED**. A unique URL that identifies the Federation Entity as defined in :ref:`trust:Federation Roles`.
+   * - **Federation Entity Public Key (JWK)**
+     - **REQUIRED**. Elliptic public key in JSON Web Key format used for signing Entity Configurations and attesting Protocol Keys, using cryptographic algorithms specified in :ref:`algorithms:Cryptographic Algorithms`.
+   * - **Certificate Signing Request (CSR)**
+     - **REQUIRED**. CSR in PKCS #10 format for X.509 certificate issuance by the Federation Authority. The CSR MUST:
+
+       - Contain **only the Federation Entity Public Key** to be certified.
+       - Be signed with the corresponding Federation Private Key.
+       - Define the certificate Subject with required attributes as specified in :ref:`trust:X.509 Certificates Issuance` for Federation Entities.
+
+.. warning::
+   Before submitting the technical onboarding request, Federation Entities MUST ensure that their ``/.well-known/openid-federation`` endpoint publishes a valid Entity Configuration (as defined in :ref:`trust:Entity Configuration`) signed with their Federation Private Key corresponding to the Federation Entity Public Key provided in the request. The Entity Configuration MUST already include Protocol Keys in the metadata with self-signed X.509 certificates in the ``x5c`` claims.
+
+
+A non-normative example of the technical information structure that Federation Entities submit during Step 1 onboarding request:
+
+.. code-block:: json
+
+  {
+    "entity_id": "https://credentials.example.gov",
+    "entity_type": "credential_issuer",
+    "jwks": {
```

Comment:
this should be an array, compliant with the data scheme traditionally used by `jwks`, as defined in IANA